src/blueprints/tv2/timeline-object-factories/tv2-sisyfos-audio-timeline-object-factory.ts

```diff
@@ -50,7 +49,7 @@ export class Tv2SisyfosAudioTimelineObjectFactory implements Tv2AudioTimelineObj
     return Math.floor(Math.random() * 10000)
   }
 
-  private createStudioMicrophonesTimelineObject(configuration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
+  public createStudioMicrophonesTimelineObject(configuration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
```

Comment:
This should continue to be private.
The method is only used in this file. It is not a part of the `Tv2AudioTimelineObjectFactory` interface. And that interface already has methods for creating Studio Mics.

Reply:
My bad. Made that change at some point, and then wound up using the `createStudioMicrophonesUpTimelineObject` method instead, by request from you i believe. Forgot to change back the access modifier. Has been fixed now.